skyportal/facility_apis/tarot.py

```diff
@@ -315,12 +330,15 @@ def submit(request, session, **kwargs):
             auth=(altdata["browser_username"], altdata["browser_password"]),
         )
 
-        if response.status_code != 200 or "New Scene Inserted" not in response.text:
-            request.status = (
-                f"rejected: status code = {response.status_code}\n\r{response.text}"
-            )
+        if "New Scene Inserted" not in response.text:
+            error_response = f"rejected: status code = {response.status_code}. "
+            if response.status_code == 200:
+                error_response += "Scene not Inserted" + (
+                    " - Hashuser not valid" if "secu_erreur" in response.text else ""
+                )
+            request.status = error_response
         else:
-            request.status = "submitted"
+            request.status = "submitted: please retrieve status"
```

Comment:
the "please" isn't really necessary, maybe just"submitted: use retrieve to validate submission" or something (not too long) that makes it clear that this is necessary to be sure that it was submitted.